src/main/java/uk/ac/ebi/eva/commons/core/models/Region.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.commons.core.models;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Region {
+
+    private String chromosome;
+    private int start;
+    private int end;
+
+    public Region(){
+        this.chromosome = null;
+        this.start = 1;
+        this.end = Integer.MAX_VALUE;
+    }
+
+    public Region(String chromosome, int start) {
+        this.chromosome = chromosome;
+        this.start = start;
+        this.end = Integer.MAX_VALUE;
+    }
+
+    public Region(String chromosome, int start, int end) {
+        this.chromosome = chromosome;
+        this.start = start;
+        this.end = end;
+    }
+
+    public Region(String region) {
+        if (region != null && !region.equals("")) {
+            if (region.indexOf(':') != -1) {
+                String[] fields = region.split("[:-]", -1);
+                if (fields.length == 3) {
+                    this.chromosome = fields[0];
+                    this.start = Integer.parseInt(fields[1]);
+                    this.end = Integer.parseInt(fields[2]);
+                } else if (fields.length == 2) {
+                    this.chromosome = fields[0];
+                    this.start = Integer.parseInt(fields[1]);
+                    this.end = Integer.MAX_VALUE;
+                }
+            } else {
+                this.chromosome = region;
+                this.start = 0;
+                this.end = Integer.MAX_VALUE;
+            }
+        }
+    }
+
+    public static Region parseRegion(String regionString) {
+        Region region = null;
+        if (regionString != null && !regionString.equals("")) {
+            if (regionString.indexOf(':') != -1) {
+                String[] fields = regionString.split("[:-]", -1);
+                if (fields.length == 3) {
+                    region = new Region(fields[0], Integer.parseInt(fields[1]), Integer.parseInt(fields[2]));
+                } else if (fields.length == 2) {
+                    region = new Region(fields[0], Integer.parseInt(fields[1]), Integer.MAX_VALUE);
```

Comment:
Could use the two arg constructor, that way it centralises the default values. e.g.:

`region = new Region(fields[0], Integer.parseInt(fields[1]));`